libs/designsystem/src/lib/components/calendar/calendar.component.scss

```diff
@@ -50,6 +50,7 @@ td {
     background-color: transparent;
     outline: none;
     border: none;
+    color: unset;
```

Comment:
You should use `inherit` when you are only "unsetting" a single (inherited) property.

- `unset` equals `inherit` for inherited properties
- `unset` equals `initial` for non-inherited properties
- The only real use case for `unset` is `all: unset;`

Reply:
```suggestion
    color: inherit;
```